core/html_api.php

```diff
@@ -283,7 +283,7 @@ function html_css_cdn_link( $p_url, $p_hash = '' ) {
  * $p_time is the number of seconds to wait before redirecting.
  * If we have handled any errors on this page return false and don't redirect.
  *
- * @param string  $p_url      The page to redirect: has to be a relative path.
+ * @param string  $p_url      The page to redirect: has to be relative to the install path (@see $g_path).
```

Comment:
It really needs to be `{@see $g_path}`, with curly braces (see [PHPDoc documentation](https://docs.phpdoc.org/3.0/guide/guides/docblocks.html#description))